src/App.jsx

```diff
@@ -1,12 +1,151 @@
+import React from 'react';
 import './App.css';
+import axios from 'axios';
+import worldmap from './assets/world_map.png';
+import continentColor from './helpers/isContinentColor.js';
+import globe from './assets/wereldbol.png';
+import roamingPopulation from './helpers/roamingpopulation.js';
+
 
 function App() {
 
+    const [countries, setCountries] = React.useState([]);
+    const [showButton, setShowButton] = React.useState(true);
+    const [error, setError] = React.useState('');
+    const [error2, setError2] = React.useState('');
```

Comment:
Als je een boolean gaat gebruiken wanneer de state aanpast, zorg er dan voor dat de state ook initialiseert met een boolean. Nu is de initialState een lege string.